treeopt/visualize.py

```diff
@@ -0,0 +1,370 @@
+import matplotlib.pyplot as plt
+from matplotlib import colors
+import matplotlib as mpl
+import numpy as np
+
+
+class Visualize:
+    def __init__(self, optimizer):
+        """
+        Initializes a visiualize Class
+        :param optimizer: Object of the TreeOpt Class containing all
+        information of the calculated metamodell
+        :type optimizer: TreeOpt Class Object
+        :return: Nothing
+        :rtype: None
+        """
+
+        self.opti_data = optimizer
+        self.num_ele = 100
+
+    def set_n(self, n):
```

Comment:
I personally prefer to not have letters as variable names or in functions. This can be called `set_number_of_points`, or even better, just `number_of_points` with the `@property` decorator.

Reply:
changed it to `set_number_of_points`